### DISCUSSION TITLE
SenseCap indicator programming with Arduino

### DISCUSSION THREAD

**Discussion by prathameshjakka** [Seeed-Solution/SenseCAP_Indicator_ESP32#27] [Q&A]
Hey could someone help me with how to setup the SenseCap indicator in Arduino. I'm confused on which display libraries to use. I want to start the display and touch panel and develop the UI for it using LVGL square line studio.

**Comment by Love4yzp** [comment]
Hey, currently @chalei has improved a way to compile [Arduino-like code](https://github.com/Seeed-Studio/wiki-documents/pull/620/files), but this part is still missing touch feature and LVGL GUI coding.
